test/ClientCDFObjectTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Acquia\ContentHubClient\test;
+
+
+use Acquia\ContentHubClient\CDF\CDFObject;
+use Acquia\ContentHubClient\CDF\ClientCDFObject;
+use Acquia\ContentHubClient\CDFAttribute;
+use PHPUnit\Framework\TestCase;
+
+class ClientCDFObjectTest extends TestCase
+{
+  /**
+   * @var ClientCDFObject
+   */
+  private $clientCdfObject;
+
+  /**
+   *
```

Comment:
@dougan88 Please fill in all PHPDoc block

Reply:
@jabxjab I'm not sure what needs to be placed there since they don't accept or return any parameters. And also they have self-explanatory names. So the only thing I can do is basically duplicate functions' names.

Reply:
@dougan88 please refer to psr-1/2
In this particular case `{@inheritdoc}` would be enough

Reply:
fixed